src/extension.ts

```diff
@@ -143,7 +143,16 @@ export async function activate(context: vscode.ExtensionContext): Promise<vscode
 					outputChannel: outputChannel
 				});
 		});
-		context.subscriptions.push(runApexGuruOnSelectedFile, runApexGuruOnCurrentFile);
+		const insertApexGuruSuggestions = vscode.commands.registerCommand(Constants.COMMAND_INCLUDE_APEX_GURU_SUGGESTIONS, async (document: vscode.TextDocument, position: vscode.Position, suggestedCode: string) => {
+			const edit = new vscode.WorkspaceEdit();
+			edit.insert(document.uri, position, suggestedCode);
+			await vscode.workspace.applyEdit(edit);
+			TelemetryService.sendCommandEvent(Constants.TELEM_SUCCESSFUL_APEX_GURU_FILE_ANALYSIS, {
```

Comment:
New telemetry to get number of times users used the suggestions ApexGuru provided, along with the number of lines that were included.